code/modules/mining/lavaland/tendril_loot.dm

```diff
@@ -852,10 +852,11 @@
 
 /datum/action/cooldown/spell/pointed/scan/is_valid_target(atom/cast_on)
 	if(!isliving(cast_on))
+		owner.balloon_alert(owner, "[cast_on] isn't a valid target!")
```

Comment:
Please avoid printing full item names into balloon alerts, they can be of an arbitrary length and are going to look weird
The user should contextually know what they just clicked on

Reply:
guh whoops